extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -7,7 +7,7 @@
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
 
-@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
```

Comment:
Why is this configuration build time? It's only used at the runtime init phase AFAICS and it's very useful to be able to override it at runtime.

Reply:
BTW, I looked at this code because I had a user asking how to override the realm at runtime so it's not a rhetorical question.